File: JavaScript/1048.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
let salario = +lines.shift()
let reajuste;
let newSalario;
if(salario <= 400.00){
    newSalario = salario * 1.15
    reajuste = salario * 0.15
    console.log(`Novo salario: ${newSalario.toFixed(2)}`)
    console.log(`Reajuste ganho: ${reajuste.toFixed(2)}`)
    console.log(`Em percentual: 15 %`)
}else if(salario <= 800.00){
    newSalario = salario * 1.12
    reajuste = salario * 0.12
    console.log(`Novo salario: ${newSalario.toFixed(2)}`)
    console.log(`Reajuste ganho: ${reajuste.toFixed(2)}`)
    console.log(`Em percentual: 12 %`)
}else if(salario <= 1200.00){
    newSalario = salario * 1.10
    reajuste = salario * 0.10
    console.log(`Novo salario: ${newSalario.toFixed(2)}`)
    console.log(`Reajuste ganho: ${reajuste.toFixed(2)}`)
    console.log(`Em percentual: 10 %`)
}else if(salario <= 2000.00){
    newSalario = salario * 1.07
    reajuste = salario * 0.07
    console.log(`Novo salario: ${newSalario.toFixed(2)}`)
    console.log(`Reajuste ganho: ${reajuste.toFixed(2)}`)
    console.log(`Em percentual: 7 %`)
}else if(salario > 2000.00){
    newSalario = salario * 1.04
    reajuste = salario * 0.04
    console.log(`Novo salario: ${newSalario.toFixed(2)}`)
    console.log(`Reajuste ganho: ${reajuste.toFixed(2)}`)
    console.log(`Em percentual: 4 %`)
}

File: Python/1048.py
sal = float(input())
newSal = 0
dif = 0
if(0 <= sal <= 400.00):
    newSal = sal+(sal*0.15)
    dif = newSal - sal
    print('Novo salario: %.2f'%newSal)
    print('Reajuste ganho: %.2f'%dif)
    print('Em percentual: 15 %')
elif(400.01 <= sal <= 800.00):
    newSal = sal+(sal*0.12)
    dif = newSal - sal
    print('Novo salario: %.2f'%newSal)
    print('Reajuste ganho: %.2f'%dif)
    print('Em percentual: 12 %')
elif(800.01 <= sal <= 1200.00):
    newSal = sal+(sal*0.10)
    dif = newSal - sal
    print('Novo salario: %.2f'%newSal)
    print('Reajuste ganho: %.2f'%dif)
    print('Em percentual: 10 %')
elif(1200.01 <= sal <= 2000.00):
    newSal = sal+(sal*0.07)
    dif = newSal - sal
    print('Novo salario: %.2f'%newSal)
    print('Reajuste ganho: %.2f'%dif)
    print('Em percentual: 7 %')
elif(sal > 2000.00):
    newSal = sal+(sal*0.04)
    dif = newSal - sal
    print('Novo salario: %.2f'%newSal)
    print('Reajuste ganho: %.2f'%dif)
    print('Em percentual: 4 %')